marlowe-playground-client/src/Simulation/BottomPanel.purs

```diff
@@ -32,25 +32,16 @@ import Simulation.State (MarloweEvent(..), _contract, _editorErrors, _editorWarn
 import Simulation.Types (Action(..), BottomPanelView(..), State, _analysisState, _bottomPanelView, _marloweState, _showBottomPanel, _showErrorDetail, isContractValid)
 import Text.Parsing.StringParser (runParser)
 import Text.Parsing.StringParser.Basic (lines)
-
-simulationBottomPanel :: State -> Array ClassName
-simulationBottomPanel state = if state ^. _showBottomPanel then [ ClassName "simulation-bottom-panel" ] else [ ClassName "simulation-bottom-panel", ClassName "collapse" ]
-
-footerPanelBg :: Boolean -> Array ClassName
-footerPanelBg display =
-  if display then
-    [ ClassName "footer-panel-bg", ClassName "expanded" ]
-  else
-    [ ClassName "footer-panel-bg" ]
+import Types (bottomPanelHeight)
 
 bottomPanel :: forall p. State -> HTML p Action
 bottomPanel state =
-  div [ classes (simulationBottomPanel state) ]
-    [ div [ class_ flex ]
+  div ([ classes [ ClassName "simulation-bottom-panel" ] ] <> [ bottomPanelHeight (state ^. _showBottomPanel) ])
```

Comment:
Once again on planet Smith, it's rainin' free monoids yo.

Reply:
sorry, just old baggage.

Reply:
high-quality old-baggage mind you